MoneyManager/BusinessLayer/Services/UserService.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using MoneyManager.BusinessLayer.DataModels;
+using MoneyManager.DataAccessLayer;
+using MoneyManager.DataAccessLayer.Repositories;
+
+namespace MoneyManager.BusinessLayer.Services
+{
+    public class UserService
+    {
+        private readonly UnitOfWork unitOfWork;
+
+        public UserService(UnitOfWork unitOfWork)
+        {
+            this.unitOfWork = unitOfWork;
+        }
+
+        public List<UserInfo> GetSortedUsersInfo()
+        {
+            return unitOfWork.Users.GetSortedUsers().Select(u => new UserInfo() { Id = u.Id, Name = u.Name, Email = u.Email }).ToList();
+        }
+
+        public UserBalance GetBalance(int userId)
+        {
+            decimal balance = unitOfWork.Transactions.GetByUserIdAndType(userId, 1)
```

Comment:
from readability perspective its better to have 2 variables instead of `LINQ expr - LINQ expr`